core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -634,6 +634,13 @@ public enum CoreError implements ScalarDbError {
       "Resuming a transaction is not allowed in single CRUD operation transactions",
       "",
       ""),
+  GET_BUILD_ERROR_OPERATION_SUPPORTED_ONLY_WHEN_NO_CONDITIONS_ARE_SPECIFIED(
+      Category.USER_ERROR,
+      "0141",
+      "This operation is supported only when no conditions are specified at all. "
```

Comment:
```suggestion
      "This operation is supported only when no conditions are specified. "
```

Reply:
Thank you for the feedback. Fixed in e9840a9, including a similar message for the scan operation.